lib/IDE/CodeCompletion.cpp

```diff
@@ -4644,6 +4644,15 @@ static void addExprKeywords(CodeCompletionResultSink &Sink) {
   AddKeyword("#dsohandle", "UnsafeMutablePointer<Void>", CodeCompletionKeywordKind::pound_dsohandle);
 }
 
+static void addAnyTypeKeyword(CodeCompletionResultSink &Sink) {
+  CodeCompletionResultBuilder Builder(
+      Sink, CodeCompletionResult::ResultKind::Keyword,
+      SemanticContextKind::None, {});
+  // pretend 'Any' is from another module
+  Builder.setKeywordKind(CodeCompletionKeywordKind::None);
+  Builder.addTextChunk("Any");
```

Comment:
Please also add a type annotation for the Any type (use `Builder.addTypeAnnotation`)